omod/src/main/java/org/openmrs/module/dhisconnector/web/controller/DHISConnectorController.java

```diff
@@ -144,6 +144,24 @@ public void saveConfig(ModelMap model, @RequestParam(value = "url", required = t
 		}
 	}
 
+	@RequestMapping(value = "/module/dhisconnector/configureServer", params = "testConfig", method = RequestMethod.POST)
+	public void testConfig(ModelMap model, WebRequest req) throws ParseException {
+
+		AdministrationService as = Context.getAdministrationService();
+		String url = as.getGlobalPropertyObject(GLOBAL_PROPERTY_URL).getPropertyValue();
+		String user = as.getGlobalPropertyObject(GLOBAL_PROPERTY_USER).getPropertyValue();
+		String pass = as.getGlobalPropertyObject(GLOBAL_PROPERTY_PASS).getPropertyValue();
+
+		if (Context.getService(DHISConnectorService.class).testDHISServerDetails(url, user, pass)) {
+			req.setAttribute(WebConstants.OPENMRS_MSG_ATTR, "Successfully Connected to DHIS2", WebRequest.SCOPE_SESSION);
+		} else {
+			req.setAttribute(WebConstants.OPENMRS_ERROR_ATTR, "Connection Failed", WebRequest.SCOPE_SESSION);
```

Comment:
same here !!

Reply:
Thanks for the suggestion.
Fixed with 8e157ab